### DISCUSSION TITLE
what is data-aw ?

### DISCUSSION THREAD

**Discussion by Montekkundan** [arthelokyo/astrowind#59] [Q&A]
Hi I had a doubt, I hope someone can clear this to me.
I saw these in html tags :` data-aw-social-share` , `data-aw-url`, `data-aw-toggle-color-scheme`
What do they do and how they implement?

**Comment by prototypa** [comment]
Hello @Montekkundan.  Good question.  I'll be extending the README.md to answer in detail.
I'll let you know soon.

So in summary, those are simple attributes used by the theme with vanilla Javascript to give minimal functionality and not tie us to a framework.  Then each person who installs the theme can choose their favorite framework (react, angular, Vue, svelte ...) and continue.

Implementation: https://github.com/onwidget/astrowind/blob/main/src/components/core/BasicScripts.astro
